### ISSUE TITLE
E3002 incorrectly reported for spotfleet tags

### ISSUE THREAD

**Issue by viraptor** [awslabs/cfn-python-lint#268]
*cfn-lint version: 0.4.2*

*Description of issue.*

I got a report:
```
E3002 Invalid Property Resources/BuildkiteSpotfleet/Properties/SpotFleetRequestConfigData/LaunchSpecifications/0/TagSpecifications/0/Tags
```
However that property should exist for TagSpecifications:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-spotfleet-spotfleetrequestconfigdata-launchspecifications.html#cfn-ec2-spotfleet-spotfleetlaunchspecification-tagspecifications
this is a list of:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-spotfleet-spotfleetrequestconfigdata-launchspecifications-tagspecifications.html

**Comment by kddejong** [comment]
@cmmeyer can you take a look at this one.  From the spec we have this. `Tags` doesn't seem to be listed.
```
"AWS::EC2::SpotFleet.SpotFleetTagSpecification": {
  "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-spotfleet-spotfleetrequestconfigdata-launchspecifications-tagspecifications.html",
  "Properties": {
    "ResourceType": {
      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-spotfleet-spotfleetrequestconfigdata-launchspecifications-tagspecifications.html#cfn-ec2-spotfleet-spotfleettagspecification-resourcetype",
      "PrimitiveType": "String",
      "Required": false,
      "UpdateType": "Mutable"
    }
  }
},
```

**Comment by rjlohan** [comment]
@kddejong this issue is accurate - there appears to be a bug in the generation of the 'Tags' property for sub-resource types. I'll dig in and sort that out. Let me know if you spot others like this with respect to Tags.
